src/main/java/com/example/trello/service/imp/BoardServiceImpl.java

```diff
@@ -0,0 +1,60 @@
+package com.example.trello.service.imp;
+
+import com.example.trello.service.exception.BoardException;
+import com.example.trello.mapper.BoardMapper;
+import com.example.trello.model.dto.modification.BoardModification;
+import com.example.trello.model.entity.BoardEntity;
+import com.example.trello.repository.BoardRepository;
+import com.example.trello.service.BoardService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class BoardServiceImpl implements BoardService {
+
+    private final BoardRepository boardRepository;
+
+    @Autowired
+    public BoardServiceImpl(final BoardRepository boardRepository) {
+        this.boardRepository = boardRepository;
+    }
+
+    public List<BoardEntity> getBoards() {
+        return boardRepository.findByOrderByNameAsc();
+    }
+
+    public BoardEntity getBoardById(final Long boardId) throws BoardException {
+        final Optional<BoardEntity> board = boardRepository.findById(boardId);
```

Comment:
instead of checking each time the optional type using `if`, you can use `.orElseThrow()` with your custom exception
```suggestion
        final BoardEntity boardEntity = boardRepository.findById(boardId).orElseThrow(() -> new BoardException(YOUR_MESSAGE));
```